connector/connector.proto

```diff
@@ -33,6 +33,7 @@ message ControlStreamRequest {
     AuthorizePeerRequest authorize_peer = 15;
     SessionRequest session = 16;
     border0.common.v1.StatsMessage stats = 17;
+    Logs logs = 18;
```

Comment:
lgtm. Could be made common (e.g. just moved to common file) in case in the future we have device log pushes.

Reply:
it's a specific connector log, with socket and plugin fields.